pkgs/by-name/cl/cloudquery/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "cloudquery";
+  version = "6.11.2";
+
+  src = fetchFromGitHub {
+    owner = "cloudquery";
+    repo = "cloudquery";
+    rev = "refs/tags/cli-v${version}";
+    hash = "sha256-zVzHPAN/UAVQF56qcoIFOqtX/Aek3+1lO2O965UYxPM=";
+  };
+
+  vendorHash = "sha256-9YvsVflHr4QSVCYCcWg1Okmncc1sNteMJEAdz92BIgI=";
+
+  CGO_ENABLED = 0;
+
+  modRoot = "cli";
+
+  # Some checks fail to download deps
+  doCheck = false;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/cloudquery/cloudquery/cli/cmd.Version=${version}"
+  ];
+
```

Comment:
```suggestion
  subPackages = [ "cmd" ];
```

Not sure

Reply:
If you use `subPackages = [ "cmd" ]`, `nixpkgs-review` throws this error: 
`go: go.mod file not found in current directory or any parent directory`

The directory with the `go.mod` file is under `cloudquery/cli/` (that's why I use the `modRoot = "cli"` statement).

Reply:
I couldn't get this to work either. There exists a  `cli/go.mod` file, but no `cli/cmd/go.mod`. I assume in order to only build the `cli/cmd` go files, there needs to be a `cli/cmd/go.mod` but I'm not too familliar with golang.